### DISCUSSION TITLE
Opening a KTON-ETH pool on Arbitrum (Unique DeFi Protocols!)

### DISCUSSION THREAD

**Discussion by johneggleston** [ktondao/KtonDAO#29] [Q&A]
I’m proposing the initiation of a KTON-ETH liquidity pool on Arbitrum. As we know, Darwinia has two native tokens: RING and KTON. Currently, RING is tradable on Arbitrum through an existing DEX pair, but KTON isn’t yet available on this network. I believe that creating a KTON-ETH pool on Arbitrum could unlock significant opportunities for KTON holders and the Darwinia ecosystem.

**Reasons for the Proposal:**

**DeFi Compatibility**: The introduction of KTON on Arbitrum will make it compatible with several innovative DeFi protocols on this network. One such protocol is the Hyperbolic Protocol ( Hyperbolicprotocol.com ), which offers a unique lending service. Users can leverage their KTON as collateral to borrow ETH, provided there is sufficient KTON-ETH liquidity on Arbitrum.

**Reduced Gas Fees**: While the Hyperbolic Protocol also operates on the Ethereum chain, the high gas fees there make it less appealing. Arbitrum’s lower fees would make the lending service more accessible and cost-effective for KTON holders.

**Enhanced Liquidity and Accessibility**: Establishing a KTON-ETH pair on Arbitrum would increase the liquidity and accessibility of KTON, potentially enhancing its market presence and utility.

**Benefits for the Darwinia ecosystem:**

**Increased Utility for KTON**: This move would expand the utility of KTON beyond its current scope, potentially increasing its value and demand.

**Broader Exposure**: By tapping into Arbitrum's growing DeFi ecosystem, Darwinia can attract new users and investors, further solidifying its position in the market.

**Community Engagement**: Engaging with innovative DeFi protocols like Hyperbolic can foster community engagement and create new use cases for our tokens.

I propose that we discuss the feasibility, potential challenges, and the steps needed to establish this KTON-ETH liquidity pool on Arbitrum. Your thoughts, suggestions, and potential concerns are highly valuable, and I look forward to a fruitful discussion.

Cheers!

**Comment by danairey** [comment]
Love this idea, please do it. We need to get as far away from eth and their gas fees. I would imagine you would use some of the treasury’s kton and also sell some of the kton into eth which would hit the kton price but long term hopefully a lot better .

**Reply by johneggleston** [reply]
Hey Danairey, thank you for your support and insight. Whilst selling some of the treasury’s KTON for ETH is a viable option for securing the ETH required for the Arbitrum pool, there are other far better options imo. 

1. Selling a portion of the RING in the Treasury to secure the ETH for the pair instead of selling the KTON.

2. Removing a small portion of the ERC20 KTON-ETH liquidity from the existing Uniswap pool to fund the Arbitrum pool.

3. Removing a small portion of the ERC20 RING-ETH liquidity and swapping the RING for ETH to fund the Liquidity pool. (This is better than option 2 imo as the existing ETH liquidity for RING is far deeper than the liquidity for KTON) 

4. A small KTON-ETH liquidity farm for Arbitrum with RING rewards. 

5. A combination of the above 4 options. 

Please let me know your thoughts on this.

**Comment by SasoLithops** [comment]
Cheers John!

Glad to read your idea. 

I have a few questions:

- on which DEX would that liquidity be? On Uniswap like Ring?
- how do we get to both KTON and ETH that will be needed to be deposited in pool? How deep do you plan the pool to be, to be long term sustainable?
- or do you plan to move liquidity from ethereum to arbitrum? Not sure if that is feasible...
- For upcomming KTON/RING liquidity on Snowswap treasury will provide incentives in form of Ring and us users will need to provide liquidity. Do you plan similar in arbitrum?

In a way I think Snowswap can become THE DEX for KTON liquidity while I would't mind having KTON contract on arbitrum as well. The native Kton can then be bridged throug Helix to Arbirtrum Kton and be used there in protocols.

**Reply by johneggleston** [reply]
Hey Saso, thank you for your interest in my idea and your great questions. 

1. Yes the liquidity would be on Uniswap just like RING. 

2. I’d say the pool should be at least 2000 KTON deep (that’s 2000 KTON on one side and an equally valued amount of ETH on the other side). The fact the liquidity is already super deep on the ETH chain and will soon also be relatively deep on the Snowswap chain means the Arbitrage trading between the different pools should keep the price fairly stable. As for sourcing the KTON and ETH for the liquidity pool, please take a look at my reply to Danairey where I cover this in detail. 

3. Moving liquidity from ETH to ARB is very feasible and I also go over various approaches to this in my reply to Danairey. 

4. Whilst this is certainly an option, I’m not entirely convinced on the sustainability of this approach. Incentivising liquidity provision from the community costs the protocol a lot of money, whilst protocol-owned liquidity actually makes the protocol a lot of money through the LP trading fee rewards.

Whilst the Snowswap DEX is great, some of the best DeFi protocols that I have seen that could serve the KTON token require there to be native ETH liquidity on Arbitrum. 

More liquidity pools = More arbitrage trading = More volume = More LP rewards and higher rankings on exchange tracking sites. 

Please let me know your thoughts. 

Thank you.

**Reply by SasoLithops** [reply]
Hey John!

Sorry for late reply. If it is easily feasible and not time consuming I am all for moving all of our liquidity from Eth to Arb. The fees will only get worse on Eth native chain and no solution to reduce them on horizon. I think Arbitrum is a good chain to nest liquidity and if it is easily feasible we can always switch back if fees and de-fi opprtunities present themselves. I don't think we have any other then CEX listings. @AurevoirXavier @hackfisher 

I also checked Hyperbolic protocol. Looks very rewarding. Maybe we could try with "smaller" ammount but I guess we could do a seperate proposal for that? What do you think?

**Reply by johneggleston** [reply]
Cheers man! Thanks for taking such serious consideration in my idea! Right now I’m in the process of trying to reach out to the guys behind Hyperbolic Protocol so we can start a Telegram group to see what we can do!

**Comment by dmytroraise** [comment]
thanks! helped

**Reply by johneggleston** [reply]
Cheers mate! Remember to upvote 😁

**Comment by hackfisher** [comment]
Anyone can do the initialize work, was told someone did this just now.

https://arbiscan.io/tx/0x5fd79e4b162a9b5ceaf3c04fa2d33d0c0e9d709306dfa78009f7f2effd58302c
https://bridge.arbitrum.io/?destinationChain=ethereum&sourceChain=arbitrum-one

https://arbiscan.io/tx/0xc3051b64f693c8e4a536f8f51727b0f8b666b39d3fb870945e733e951c37c289

But need more liquidity from the community, and might be a good idea to request helix for 3rd bridge support from Darwinia(if someone is going to register a liquidity node relayer on Helix for this)

**Reply by johneggleston** [reply]
Awesome! Honoured that you guys like my idea.

**Comment by Berri3910** [comment]
An interesting solution and rationale for creating such a liquidity pool!
